### PR TITLE
287: Add Blogs section to Community page

### PR DESCRIPTION
**Pull request by juhalehtonen**
# Changes

Addresses issue #287 by adding community blog links to the Community page. 

Still a work in progress due to missing copy. How should we proceed with the descriptions. Shall we write our own?

# Todo

- [x] I acknowledge my contribution to the website does not assert comparative or superlative differences of one product, project, company or individual over another.
- [x] PR needs finalized copy

### CODE REVIEW

**Comment by starbelly** [comment]
@ juhalehtonen Awesome stuff! I will try to review this some time today or tomorrow. ❤️

File: priv/data/community/blogs.exs [hunk 1]
```diff
@@ -0,0 +1,82 @@
+[
+  %{
+    name: "Dashbit",
+    link: "https://dashbit.co/blog/",
+    icon: "/images/community/blogs/dashbit.png",
+    logo: nil,
+    about:
+      "Dashbit copy."
```
Comment (josevalim) [inline priv/data/community/blogs.exs:8]:
```suggestion
      "Maintainers and contributors to many Elixir projects, such as Ecto, Broadway, LiveView, and Elixir itself."
```

**Comment by starbelly** [comment]
@juhalehtonen Right, per @josevalim jumping in... I'm starting to reach out to people who run these blogs, like I did with the languages section, in order to get what they would like to say about their blog. We may not be able to contact everyone, but we should make a best effort IMO.

File: priv/data/community/blogs.exs (hunk 1 above)
Comment (juhalehtonen) [reply priv/data/community/blogs.exs]: Wonderful, thank you @josevalim 💜

**Comment by juhalehtonen** [comment]
> 
> 
> @juhalehtonen Right, per @josevalim jumping in... I'm starting to reach out to people who run these blogs, like I did with the languages section, in order to get what they would like to say about their blog. We may not be able to contact everyone, but we should make a best effort IMO.

Great, thank you! :) I had the same thought before, but hadn't gotten around to acting on that.

File: priv/data/community/blogs.exs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    about:
+      "Planet Erlang is a meta blog that collects posts from the blogs of various Erlang developers and contributors."
+  },
+  %{
+    name: "Elixir Radar",
+    link: "https://elixir-radar.com/",
+    icon: nil,
+    logo: nil,
+    about:
+      "Keep updated with what's happening in the Elixir community."
```
Comment (hugobarauna) [inline priv/data/community/blogs.exs:56]: Hi, Hugo from Elixir Radar here! 🙂 

Can you change the about value to:

```
Elixir Radar is a curated content weekly Elixir newsletter and a job board for Elixir developers.
```

"

File: priv/data/community/blogs.exs [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    icon: nil,
+    logo: nil,
+    about:
+      "Planet Erlang is a meta blog that collects posts from the blogs of various Erlang developers and contributors."
+  },
+  %{
+    name: "Elixir Radar",
+    link: "https://elixir-radar.com/",
+    icon: nil,
+    logo: nil,
```
Comment (hugobarauna) [inline priv/data/community/blogs.exs:54]: Do you need our logo?

Unfortunately, we don't an icon. :(

File: priv/data/community/blogs.exs (hunk 2 above)
Comment (juhalehtonen) [reply priv/data/community/blogs.exs]: Thank you very much Hugo, both for the text and for Elixir Radar! :) I've updated the copy.

File: priv/data/community/blogs.exs (hunk 3 above)
Comment (juhalehtonen) [reply priv/data/community/blogs.exs:54]: If you have a logo that we could use here, that'd be great! I looked around for icons but as you say, didn't find any :) I think the current "logos" are used the same way Elixir's logo is used here: https://erlef.org/community/.

As an aside to @starbelly,  I think this use of logos will probably need some extra love at some point. We'll want to ensure the names don't need to be removed (as in Elixir's case) for better accessibility and SEO :) I can look at that in a separate PR.

Comment (starbelly) [reply priv/data/community/blogs.exs:54]: Logo is fine. We merely make a distinction (which is fuzzy) between icon and logo for css classes iirc. In this sub-domain an icon is said to be an image without words, a logo is an image with words. In reality of course the distinction between logo and icon is not exactly that, but it helps here :)

Comment (starbelly) [reply priv/data/community/blogs.exs:54]: @joaquinalcerro I agree 💯 . This app needs ❤️ all over the place! 😁

Comment (joaquinalcerro) [reply priv/data/community/blogs.exs:54]: Hi @starbelly sure thing, I will check the open issues to try addressing some of them. You guys have done a amazing job so far...

**Comment by juhalehtonen** [comment]
I went through the list of items I had copied off of https://github.com/erlef/marketing-wg/wiki/Community-Sources in order to write up copy for the still-missing items, and started to wonder if some of the entries I have in this PR make sense to include here. 

For example, despite their quality and usefulness, `Changelog`, `Elixir Weekly` and `Elixir Status` are three sites which might or might not be considered "blogs" in the traditional sense of the word, and some others might be on the verge of not quite being that either. 

Do we still wish to introduce them under the Blog heading on erlef.org, or should we potentially place them under a separate heading (and/or drop them altogether, if no fitting category is planned)? What's your take @starbelly ?

**Comment by starbelly** [comment]
> I went through the list of items I had copied off of https://github.com/erlef/marketing-wg/wiki/Community-Sources in order to write up copy for the still-missing items, and started to wonder if some of the entries I have in this PR make sense to include here.
> 
> For example, despite their quality and usefulness, `Changelog`, `Elixir Weekly` and `Elixir Status` are three sites which might or might not be considered "blogs" in the traditional sense of the word, and some others might be on the verge of not quite being that either.
> 
> Do we still wish to introduce them under the Blog heading on erlef.org, or should we potentially place them under a separate heading (and/or drop them altogether, if no fitting category is planned)? What's your take @starbelly ?


Yeah, it's fuzzy. We could have a podcasts section or we could call this section "Blogs and Podcasts". I think we shouldn't let good enough be the enemy of perfect here though, so I'd be happy with either myself, we can always change it later :)

**Comment by starbelly** [comment]
On another note we missing a blog in the list right now and need to update the marketing wiki as well : https://medium.com/erlang-battleground

@elbrujohalcon would you like to provide a description for the blog for this list?

**Comment by starbelly** [comment]
On other one while I'm thinking about... @ferd Would you like to make a suggestion for adding ferd.ca to the roster?

**Comment by rrrene** [comment]
@starbelly I feel very honored that ElixirStatus and ElixirWeekly are considered here 💜 

The two sites could be described like this:

- ElixirStatus is a community where Elixir creators post their newest blog posts, project updates, podcast episodes and videos.
- ElixirWeekly is focussing on the very latest creations of the community in one weekly email.

**Comment by stevenproctor** [comment]
@starbelly The Planet Erlang description works for me.  There is no logo/icon for the site as it stands.

Also, feel free to browse the list I have collected so far to use as an additional source of blogs to reach out to if you all would like:  https://github.com/stevenproctor/planet-erlang/blob/master/erlang/config.ini#L89

I am sure I will have to come back here and see what you all have added to see what I am missing from the Erlang side. (I had requests from people not wanting Elixir at one point, but community consensus may have changed since then...)

**Comment by juhalehtonen** [comment]
I've now added a description to each of the blogs (many thanks to the people who contributed here 💜!), as well as an intro paragraph about the blog section itself. Would be great if you @starbelly find time to review them, as I'm not familiar with the foundation's tone of voice and such :)

**Review by starbelly — APPROVED** [review]

**Comment by starbelly** [comment]
@juhalehtonen Perfect! Thank you for knocking this one out!  ❤️💛💚💙💜❤️💛💚💙💜❤️💛💚💙💜❤️💛💚💙💜

**Pull request merged**